tools/tool.py

```diff
@@ -268,16 +305,16 @@ def UpdateTableAlive(self, database, name, pid, endpoint, is_alive) -> Status:
             return Status()
         return Status(-1, "update table alive failed")
 
-    def ChangeLeader(self, database, name, pid, sync = False) -> Status:
+    def ChangeLeader(self, database, name, pid, endpoint = "auto", sync = False) -> Status:
         cmd = list(self.ns_base_cmd)
-        cmd.append("--cmd=changeleader {} {} auto".format(name, pid))
+        cmd.append("--cmd=changeleader {} {} {}".format(name, pid, endpoint))
         cmd.append("--database=" + database)
         status, output = self.RunWithRetuncode(cmd)
         if status.OK() and sync and not self.WaitingOP(database, name, pid).OK():
             return Status(-1, "changer leader failed")
         return status
 
-    def ShowOpStatus(self, database, name, pid = '') -> tuple([Status, List]):
+    def ShowOpStatus(self, database, name = '', pid = '') -> tuple([Status, List]):
```

Comment:
`showopstatus` and `showtablestatus` need not to disable and recover `autofailover`